src/__tests__/TextInput.test.tsx

```diff
@@ -98,22 +98,24 @@ describe('TextInput', () => {
     const handleAction = jest.fn()
     expect(
       render(
-        <TextInput
-          name="search"
-          placeholder={'Search'}
-          trailingAction={<TextInput.Action onClick={handleAction} icon={SearchIcon} aria-label="iconLabel" />}
-        />
+        <SSRProvider>
```

Comment:
notes for reviewer: Wrapped in `SSRProvider` because trailingAction includes a tooltip with aria-labelledby